<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadPlanDialog</class>
 <widget class="QDialog" name="LoadPlanDialog">
  <property name="windowTitle">
   <string>Avaa kaava</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">

   <!-- Connection selection section -->
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Valitse tietokantayhteys:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="connectionComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>

   <item>
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
    </spacer>
   </item>

   <item>
    <widget class="QLabel" name="label_search">
     <property name="text">
      <string>Etsi kaavoja:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="searchLineEdit">
     <property name="placeholderText">
      <string>Etsi kaavoja...</string>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>

   <item>
    <spacer name="spacer2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
    </spacer>
   </item>

   <!-- Table view for displaying plans -->
   <item>
    <widget class="QLabel" name="label_plan_table">
     <property name="text">
      <string>Kaavat:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="planTableView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
   </widget>
   </item>

   <item>
    <spacer name="spacer3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
    </spacer>
   </item>

   <!-- OK button -->
   <item>
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
